0033-relationship-between-protocols/0033-relationship-between-protocols.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Relationship between Protocols
+draft: 1
+---
+
+# Relationship between Protocols
+
+## Prerequisites
+This document assumes the reader is familiar with the following document:
+
+- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
+
+## Terminology
+- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver.
+- A **payment** is sending value from one to another but it doesn't mean settled necessarily that is explained below. In the Interledger network, a payment could mean a **node-to-node** payment or an **end-to-end** payment.
+  - A **node-to-node** payment means the intention of transferring value of a node (a connector, an endpoint sender, or an endpoint receiver) to another node which is directly connected to the sender node.
+  - An **end-to-end** payment means the intention of transferring value of an endpoint sender to an endpoint receiver.
+- A **settlement** is ensuring that one has the paid value. For instance, executing a transaction on a certain ledger that the payee has an account on.
+  - A **node-to-node** payment is settled by means that the two nodes agreed upon. It is usually a ledger that tracks the amount that both nodes have.
+  - An **end-to-end** payment is settled through a chain of settlements of intermediaries above.
```

Comment:
Nitpick: Wouldn't an end-to-end payment be settled from the perspective of the sender and receiver as soon as the sender gets the fulfillment? Or is it not yet settled because some people in the chain may have outstanding obligations? @adrianhopebailie

Reply:
I think...getting a fulfillment doesn't mean a settlement, it is the proof that the payment is approved to be settled. Also when we can say "the end-to-end payment is settled" is difficult, but if we expect the settlement means the `entire settlement` of the chain, all the nodes should be settled.

Anyway, I fixed a bit to make it more clear.